File: 351004/Beliaev/restapp/discussion/src/main/java/com/example/discussion/model/Note.java
package com.example.discussion.model;

import lombok.Data;
import org.springframework.data.cassandra.core.mapping.Column;
import org.springframework.data.cassandra.core.mapping.PrimaryKey;
import org.springframework.data.cassandra.core.mapping.Table;

@Data
@Table("tbl_note")
public class Note {
    @PrimaryKey
    private Long id;

    @Column("article_id")
    private Long articleId;

    private String content;

    private String state; // PENDING, APPROVE, DECLINE
}

File: 351004/Narivonchik/rest1/DiscussionModule/src/models/Note.cs
using Cassandra.Mapping.Attributes;

namespace DiscussionModule.models;

[Table("tbl_notes")]
public class Note
{
    [PartitionKey]
    [Column("id")]
    public long Id { get; set; }

    [Column("news_id")]
    public long NewsId { get; set; }

    [Column("content")]
    public string Content { get; set; } = string.Empty;

    [Column("country")]
    public string Country { get; set; } = string.Empty;
    
    [Column("state")]
    public string State { get; set; } = "PENDING"; // PENDING, APPROVE, DECLINE
}